bobyqa/funHandleWrap.m

```diff
@@ -0,0 +1,15 @@
+function [ y ] = funHandleWrap(x, calfun)
+% Wrapper to compute objective function value. Currently used global
+% variable in bobyqa.m, the argument calfun is not used.
```

Comment:
"Currently used global variable in bobyqa.m," ? Don't understand.

Reply:
"currently uses global variable in bobyqa.m": means that currently the objective function here comes from a global variable defined in bobyqa.m. This is not-optimal design, but it is difficult/impossible to refer to a non-globally defined matlab function from within mex files. I will change the doc.